rio/srcrobot/robot_container.py

```diff
@@ -56,85 +60,67 @@ class RobotContainer:
         self.climbUp = self.operator.povLeft()
         self.climbDown = self.operator.povRight()
-        # self.climbUpM = self.operator.leftStick()
         self.configureButtonBindings()
 
         NamedCommands.registerCommand("RevShooter", self.s_Shooter.shoot().withTimeout(2.0).withName("AutoRevShooter"))
         NamedCommands.registerCommand("Shoot", self.s_Indexer.indexerShoot().withTimeout(1.0).withName("AutoShoot"))
         NamedCommands.registerCommand("ShooterOff", InstantCommand(self.s_Shooter.brake, self.s_Shooter).withName("AutoShooterBrake"))
         NamedCommands.registerCommand("IndexerIntake", self.s_Indexer.indexerIntakeOnce().withName("AutoIndexerIntake"))
-        NamedCommands.registerCommand("IndexerOff", self.s_Indexer.stopIndexer().withName("AutoIndexerOff"))
+        NamedCommands.registerCommand("IndexerOff", self.s_Indexer.instantStop().withName("AutoIndexerOff"))
         NamedCommands.registerCommand("IntakeOn", self.s_Intake.intakeOnce().withName("AutoIntakeOn"))
-        NamedCommands.registerCommand("IntakeOff", self.s_Intake.stopIntake().withName("AutoIntakeOff"))
-        # NamedCommands.registerCommand("Command ")
+        NamedCommands.registerCommand("IntakeOff", self.s_Intake.instantStop().withName("AutoIntakeOff"))
 
         self.auton_selector = SendableChooser()
         self.auton_selector.setDefaultOption("Straight Auto", PathPlannerAutoRunner("StraightAuto", self.s_Swerve).getCommand())
         self.auton_selector.addOption("RightSubwooferTaxiAuto", PathPlannerAutoRunner("RightSubwooferTaxiAuto", self.s_Swerve).getCommand())
         self.auton_selector.addOption("CenterSubwoofer2PieceAuto", PathPlannerAutoRunner("CenterSubwoofer2Piece", self.s_Swerve).getCommand())
 
-        SmartDashboard.putData("Auton Selector", self.auton_selector)
-
-        SmartDashboard.putBoolean("Field Centric", self.getFieldOriented())
-        SmartDashboard.putBoolean("Zero Gyro", self.zeroGyro.getAsBoolean())
-        SmartDashboard.putData("Swerve Subsystem", self.s_Swerve)
-        SmartDashboard.putData("Intake Sub", self.s_Intake)
-        SmartDashboard.putData("Indexer Sub", self.s_Indexer)
-        SmartDashboard.putData("Shooter Sub", self.s_Shooter)
-        SmartDashboard.putNumber("FLYWHEEL TARGET", self.s_Shooter.getTargetVelocity())
+        Shuffleboard.getTab("Autonomous").add("Auton Selector", self.auton_selector)
+        Shuffleboard.getTab("Teleoperated").add("Swerve Subsystem", self.s_Swerve)
+        Shuffleboard.getTab("Teleoperated").add("Intake Sub", self.s_Intake)
+        Shuffleboard.getTab("Teleoperated").add("Indexer Sub", self.s_Indexer)
+        Shuffleboard.getTab("Teleoperated").add("Shooter Sub", self.s_Shooter)
+        Shuffleboard.getTab("Teleoperated").add("Arm Sub", self.s_Arm)
+        Shuffleboard.getTab("Teleoperated").add("Climber Sub", self.s_Climber)
+        Shuffleboard.getTab("Teleoperated").addString("QUEUED SHOT", self.getQueuedShot)
```

Comment:
Glad to see this on the dashboard. The driver should only ever look at the dashboard for the camera to pick up a note at the far end. This shot setting may be the only reason for the operator to look, to verify the shot is set as expected. BTW, can we increase the size of the font on the dashboard for this value?

Reply:
As far as my minimal research goes I see that the font size is pretty much constant. It lets you set the title size on the dashboard itself but not the actual data at least as far as the python api goes. There may be a way to do it on the dashboard gui itself though.